### DISCUSSION TITLE
Getting started with Elasticsearch | ashish.one

### DISCUSSION THREAD

**Discussion by giscus** [ashishtiwari1993/ashish.one#19] [General]
# Getting started with Elasticsearch | ashish.one

Sample Queries for Elasticsearch Workshop CRUD # Insert POST meetup/_doc/ { "name":"Ashish Tiwari" } # Insert with id POST meetup/_doc/1 { "name":"Ashish Tiwari" } # Search GET meetup/_search # Update POST meetup/_doc/1 { "name":"Ashish", "company":"elastic", "address":"Navi Mumbai kharghar", "skills":{ "language":["php","java","node"], "database":["mysql","mongodb"], "search":"elasticsearch" } } # search with query GET meetup/_search { "query": { "match": { "address": "navi" } } } # delete DELETE meetup BULK POST _bulk {"index":{"_index":"meetup"}} {"user_id":1,"first_name":"Yvonne","last_name":"Willmott","email":"ywillmott0@live.com","gender":"Female","street_address":"38 Helena Avenue","ip_address":"104.

https://ashish.one/talks/ws-es/

**Comment by vaibhav-1508** [comment]
Very insightful
